code/qcommon/q_platform.h

```diff
@@ -124,9 +124,9 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 
 #if defined( _M_IX86 ) || defined( __i386__ )
 #define ARCH_STRING "x86"
+#define HAVE_VM_COMPILED
 #elif defined(__arm__) || defined(_M_ARM)
 #define ARCH_STRING "arm"
-#define NO_VM_COMPILED
```

Comment:
I'm not sure why the existing armv7l JIT isn't used on Windows, but perhaps it doesn't work there. I'm preserving existing functionality rather than trying to improve a platform I can't test.